src/main/java/com/example/bank/bankTransfer/transfer/TransferService.java

```diff
@@ -69,18 +74,38 @@ BigDecimal exchangeAmount(Currency fromAccountCurrency, Currency toAccountCurren
     }
 
     public void transfer(Account fromAccount, Account toAccount, BigDecimal amount) {
+        TransferHistory fromAccountTransferHistory =
+                transferHistoryService.buildFromAccountTransferHistory(fromAccount, amount, toAccount);
+        TransferHistory toAccountTransferHistory =
+                transferHistoryService.buildToAccountTransferHistory(toAccount, amount, fromAccount);
```

Comment:
te zwroty account to account from tez glupio brzmia. 
Wez uzyj slowa odbiorca nadawca